docs/game-mechanics/behaviors/basic-attack.rst

```diff
@@ -7,33 +7,48 @@ Parameters
 ----------
 
 .. list-table ::
-   :widths: 15
+   :widths: 15 30
    :header-rows: 1
 
    * - Name
+     - Description
    * - dir_angle_xz
+     - The direction to modify the attackers angle by?  Currently unused in Darkflame Universe.
    * - dir_angle_y
+     - The direction to modify the attackers angle by?  Currently unused in Darkflame Universe.
    * - dir_force
+     - The amount of force to apply towards angle specified in :samp:`dir_angle_xz` or :samp:`dir_angle_y`?  Currently unused in Darkflame Universe.
    * - dont_apply_immune
+     - Whether or not to apply immunity to the caster of the behavior.  Currently unused in Darkflame Universe.
    * - max damage
+     - The maximum amount of damage to be dealt.
    * - min damage
+     - The minimum amount of damage to be dealt.
    * - on_fail_armor
+     - The behavior to use on failure due to armor.  Currently unused in Darkflame Universe.
    * - on_fail_blocked
+     - The behavior to use on failure due to the attack being blocked.  Currently unused in Darkflame Universe.
    * - on_fail_immune
+     - The behavior to use on failure due to the target being immune.  Currently unused in Darkflame Universe.
    * - on_success
+     - The behavior to use on success of the attack.
    * - radius
+     - The radius of the attack.  This is likely deprecated as only `one behavior <https://explorer.lu/skills/69>`_ currently uses this.  Currently unused in Darkflame Universe.
    * - use_caster_velocity
+     - Whether or not to use the casters velocity.  Currently unused in Darkflame Universe.
    * - velocity_multiplier
+     - Value to multiply velocity by when attacking.  Currently unused in Darkflame Universe.
 
 BitStream Serialization
 -----------------------
 
-align to byte boundary (don’t ask me why, this (and the “padding” below) is completely pointless)
+align to byte boundary (don’t ask me why, this (and the "padding" below) is completely pointless)
 
-| **[u16]** - “padding”
-| **[bit]** - ???, always False?
-| **[bit]** - ???, always False?
-| **[bit]** - ???, always True?
-| **[u32]** - ???
-| **[u32]** - damage
-| **[bit]** - ???, maybe whether the attack is part of an Area of Effect attack?
+| **[u16]** - This is just padding.
+| **[bit]** - True if the attack was blocked.  False otherwise.  As of February 11, 2022, this value is always serialized as false.
```

Comment:
That should be marked as DLU specific. And I don't think it's true for SyncSkills, just for server generated skill calculations. Maybe put these in a separate section

Reply:
ah ok - will update this in the morning then.  I added the addendum based on calculate and didn't update it after our discussion in Discord.

Reply:
Not quite. Keep the general heading and add a new one for DLU specific notes is what I meant